pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/impl/GenericBrokerHostUsageImpl.java

```diff
@@ -62,8 +67,10 @@ public SystemResourceUsage getBrokerHostUsage() {
     }
 
     private void checkCpuLoad() {
-        cpuUsageSum += systemBean.getSystemCpuLoad();
-        cpuUsageCount++;
+        synchronized (this) {
```

Comment:
nit: what about moving the `synchronized` keyword in the method declaration ?

Reply:
Great point. I think that makes more sense. I'll update it when I'm back online.

Reply:
@michaeljmarshall if you happen to make further changes, it would be useful to rebase the PR on latest master branch changes. Good work! Thank you

Reply:
Done.